quantum/pointing_device_drivers.c

```diff
@@ -244,6 +244,44 @@ const pointing_device_driver_t pointing_device_driver = {
     .set_cpi    = pmw3360_set_cpi,
     .get_cpi    = pmw3360_get_cpi
 };
+
+// clang-format on
+#elif defined(POINTING_DEVICE_DRIVER_pmw3389)
+
+
+report_mouse_t pmw3389_get_report(report_mouse_t mouse_report) {
+    report_pmw3389_t data        = pmw3389_read_burst();
+    static uint16_t  MotionStart = 0;  // Timer for accel, 0 is resting state
+
+    if (data.isOnSurface && data.isMotion) {
+        // Reset timer if stopped moving
+        if (!data.isMotion) {
+            if (MotionStart != 0) MotionStart = 0;
+            return mouse_report;
+        }
+
+        // Set timer if new motion
+        if ((MotionStart == 0) && data.isMotion) {
+#    ifdef CONSOLE_ENABLE
+            if (debug_mouse) dprintf("Starting motion.\n");
+#    endif
+            MotionStart = timer_read();
+        }
+        mouse_report.x = constrain_hid(data.dx);
+        mouse_report.y = constrain_hid(data.dy);
+    }
+
+    return mouse_report;
+}
+
+// clang-format off
+const pointing_device_driver_t pointing_device_driver = {
+    .init       = pmw3389_init,
```

Comment:
Sorry, this change should be reverted, or the pmw3389_init function changed to void, rather than bool. (ideally, reverted).

```suggestion
    .init       = init,
```